### PR TITLE
Fork kops-aws-prow job

### PR DESCRIPTION
**Pull request by krzyzacy**
(trying out)
/hold

### CODE REVIEW

**Comment by BenTheElder** [comment]
this will need kubernetes-security or disabling that presubmit. if we want to do the latter we should discuss it somewhere.

**Comment by krzyzacy** [comment]
(Rewriting k-s/k presubmit to make it less dumb)

**Comment by krzyzacy** [comment]
untouched k-s/k presubmit, simply copy-pasted for now

File: prow/config/jobs_test.go [hunk 1]
```diff
@@ -49,6 +49,28 @@ func flattenJobs(jobs []Presubmit) []Presubmit {
...
+// Returns if two brancher has overlapping branches
+func CheckOverlapBrancher(b1, b2 Brancher) bool {
+	if b1.RunsAgainstAllBranch() || b2.RunsAgainstAllBranch() {
+		return true
+	}
+
+	overlap := []string{}
+	for _, run1 := range b1.Branches {
+		if b2.RunsAgainstBranch(run1) {
+			overlap = append(overlap, run1)
```
Comment (BenTheElder) [inline prow/config/jobs_test.go]: just return true?

File: prow/config/jobs.go [hunk 2]
```diff
@@ -103,9 +103,13 @@ type Brancher struct {
 	Branches []string `json:"branches"`
 }
 
+func (br Brancher) RunsAgainstAllBranch() bool {
+	return len(br.SkipBranches) == 0 && len(br.Branches) == 0
+}
+
 func (br Brancher) RunsAgainstBranch(branch string) bool {
 	// Favor SkipBranches over Branches
```
Comment (BenTheElder) [inline prow/config/jobs.go]: stale comment?

File: prow/config/jobs_test.go (hunk 1 above)
Comment (krzyzacy) [reply prow/config/jobs_test.go]: done

File: prow/config/jobs.go (hunk 2 above)
Comment (krzyzacy) [reply prow/config/jobs.go]: whoops, moved down for where this comment actually belongs to

**Comment by krzyzacy** [comment]
:snowflake: Travis :cry:

**Comment by BenTheElder** [comment]
Noting that the branch name is terrible and confusing :-) /s
Hopefully this is fine and we can start branching all the jobs. I'm excited to get master onto bazel 0.6 :smile:

**Comment by BenTheElder** [comment]
heh that's a goofy response message:

> @krzyzacy: I updated Prow config for you!
>...
> In response to this:
> > (trying out)
> > /hold
>
>...

**Pull request merged**